lib/components/viewers/stop-viewer.js

```diff
@@ -228,13 +237,26 @@ class StopViewer extends Component {
               onClick={this._toggleScheduleView}
               style={{ fontSize: 'small' }}
             >
-              <Icon type={scheduleView ? 'clock-o' : 'calendar'} withSpace />
+              <Icon
+                type={isShowingSchedule ? 'clock-o' : 'calendar'}
+                withSpace
+              />
               <FormattedMessage
                 id="components.StopViewer.viewTypeBtnText"
-                values={{ scheduleView }}
+                values={{ scheduleView: isShowingSchedule }}
               />
             </button>
           )}
+          {isShowingSchedule && (
+            <input
```

Comment:
I didn't realize this was changeable! I tried adding a calendar icon but it messed with the formatting too much. I'll open an issue

Reply:
We are mostly at the mercy of each browser for that. Or we could use a react-bootstrap component.

Reply:
Yeah I've changed the cursor for now, but I think when/if we re-design the stop viewer this is something to keep in mind